ur3: #namespace

    #This would have to be changed if not running in my VM
    ros_ws_abspath: "/home/adam/catkin_ws"

    n_actions: 6
   
   
    #qlearn parameters
    alpha: 0.5
    gamma: 0.9
    epsilon: 0.1

    #alpha: 0.2
    #gamma: 0.9
    #epsilon: 0.1
    
    #alpha: 0.1
    #gamma: 0.7
    #epsilon: 0.9

    
    epsilon_discount: 0.999

    #Number if iterations
    nepisodes: 500
    #Number of steps per iteration
    nsteps: 500
    

    init_pos: 0.0 # Position in which the base will start
    wait_time: 0.1 # Time to wait in the reset phases